README.md

```diff
@@ -57,10 +38,134 @@ The following commands are for Ubuntu Server. However, equivalent commands shoul
    * Debug Build: `cargo build`
    * Release Build: `cargo build --release`
    * Run Tests: `cargo test`
-   * Run Server: `cargo run --bin modelardbd path_to_data_folder`
+   * Run Server: `cargo run --bin modelardbd path_to_local_data_folder`
    * Run Client: `cargo run --bin modelardb [server_address] [query_file]`
 5. Move `modelardbd` and `modelardb` from the `target` directory to any directory.
 
+## Usage
+`modelardbd` supports two execution modes *edge* and *cloud*. For storage, `modelardbd` uses local storage and an Amazon S3-compatible object store (optional in edge mode). The execution mode dictates where queries are executed, when `modelardbd` is deployed in edge mode it executes queries against local storage and when it is deployed in cloud mode it executes queries against the object store. For both deployment modes, `modelardbd` automatically compresses the ingested time series using multiple different types of models and continuously transfers this compressed representation from local storage to the object store. Be aware that sharing metadata between multiple instances of `modelardbd` is currently under development, thus only the instance of `modelardbd` that ingested a time series can currently query it.
```

Comment:
(1) I am not sure why the paragraph being written as a single line is problems since it is [rendered as paragraph](https://github.com/ModelarData/ModelarDB-RS/tree/doc/add-usage-to-readme#usage)? Do you think we should add a newline after each punctuation to make it more readable in a text editor?

(2) I have replaced the comma with a punctuation.

Reply:
Perhaps I'm just old-fashioned :-) I agree that it looks fine when rendered as a paragraph, but it also nice to be able to easily read it in a terminal (and it would still be rendered nicely as a paragraph). But it is not a big deal

Reply:
I tried a few ways to format the document with a 80 char bound, but it just made the text less readable in my opinion. Thus I have left the text in paragraphs with an unbound line length.

Reply:
ok